Documentation/usage/README.md

```diff
@@ -1,5 +1,35 @@
 # Using Delve
 
-You can invoke Delve in multiple ways, depending on your usage needs. Delve makes every attempt to be user-friendly, ensuring the user has to do the least amount of work possible to begin debugging their program.
+You can invoke Delve in [multiple ways](dlv.md), depending on your usage needs. Delve makes every attempt to be user-friendly, ensuring the user has to do the least amount of work possible to begin debugging their program.
 
-Refer to the [main usage document](dlv.md) to further explore commands.
+The [available commands](dlv.md) can be grouped into the following categories:
+
+*  Specify target and start debugging with a [terminal interface](../cli/README.md):
+   * [dlv debug [package]](dlv_debug.md)
+   * [dlv test [package]](dlv_test.md)
+   * [dlv exec \<exe\>](dlv_exec.md)
+   * [dlv attach \<pid\>](dlv_attach.md)
+   * [dlv core \<exe\> \<core\>](dlv_core.md)
+   * [dlv replay \<rr trace\> ](dlv_replay.md)
+   * [dlv trace [package] \<regexp\>](dlv_trace.md)
+* Start a headless backend server and connect with a [frontend client](../EditorIntegration.md):
+   * [dlv **--headless** \<command\> \<target\> \<args\> ](../api/ClientHowto.md#spawning-the-backend)
```

Comment:
How strongly do you feel about regrouping? Tracing can be its own thing, but otherwise my main goal was to separate the all-in-one debugging with dlv terminal from the usage mode of connecting external clients. Your grouping looses that clear distinction.

Reply:
I moved trace to its own line since it doesn't use the client and added a comment about no headless.

Reply:
It's ok to not regroup, but I still think it should say what `<command>` is here, no?

Reply:
Done